Source/VCGeneration/Counterexample.cs

```diff
@@ -508,6 +508,7 @@ public class CallCounterexample : Counterexample
   {
     public CallCmd FailingCall;
     public Requires FailingRequires;
+    public AssertRequiresCmd UnderlyingAssert;
```

Comment:
Would `FailingAssert` be a more specific name? I'm not able to get any information out of the term `Underlying`

Reply:
Makes sense